### PR TITLE
feat: fix pluralization in vue-i18n

### PR DESCRIPTION
**Pull request by asbiin**
This adds pluralization form for vue-i18n
See https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument

### CODE REVIEW

File: resources/assets/js/pluralization.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  number = Math.abs(number);
+  number = (number == 1) ? 0 : (((number >= 2) && (number <= 4)) ? 1 : 2);
+  return Math.min(number, choicesLength - 1);
+}
+function pluralD (number, choicesLength) {
+  number = Math.abs(number);
+  number = ((number % 10 == 1) && (number % 100 != 11)) ? 0 : (((number % 10 >= 2) && (number % 10 <= 4) && ((number % 100 < 10) || (number % 100 >= 20))) ? 1 : 2);
+  return Math.min(number, choicesLength - 1);
+}
+function pluralE (number, choicesLength) {
```
Comment (yarons) [inline resources/assets/js/pluralization.js]: This is the plural form for Hebrew, needs conversion to the correct form:
`nplurals=4; plural=(n == 1) ? 0 : ((n == 2) ? 1 : ((n > 10 && n % 10 == 0) ? 2 : 3))`

**Comment by yarons** [comment]
Requires some additional adjustments so it'll work with Hebrew.

**Comment by asbiin** [comment]
Thank you @yarons! Can you provide a PR for it? I took the function from Laravel... If there is a problem, maybe we could fix it too 😉

File: resources/assets/js/pluralization.js (hunk 1 above)
Comment (asbiin) [reply resources/assets/js/pluralization.js]: @yarons Could you check these source too?
- http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html
- https://github.com/laravel/framework/blob/master/src/Illuminate/Translation/MessageSelector.php

**Pull request merged**